src/templates/forum/index.html

```diff
@@ -0,0 +1,6 @@
+{% extends 'machina/forum/index.html' %}
+
+{% block content %}
+  {{ block.super }}
+  <a class="btn btn-primary" href="{% url 'forums:forum-create' %}">Create new forum</a>
```

Comment:
The placement of the button is somewhat awkward here from a UX perspective. Perhaps putting it on the top right - ish area would make more sense.